x-pack/plugins/observability/public/hooks/slo/use_create_slo.ts

```diff
@@ -66,6 +52,27 @@ export function useCreateSlo() {
         // Return a context object with the snapshotted value
         return { previousSloList: data };
       },
+      onSuccess: (_data, { slo }) => {
```

Comment:
Does the order between `onSuccess`, `onError`, and `onMutate`  have an effect?

Reply:
Not at all, but from the lifecycle of RQ, onMutate is called first, then onSuccess or onError. I just wanted to have this reflected when reading the code